app/src/androidTest/java/org/mozilla/tv/firefox/FirefoxTestRunner.kt

```diff
@@ -40,6 +45,14 @@ class FirefoxTestRunner : AndroidJUnitRunner() {
         fakeServiceLocator.swapIfNotNull()
 
         super.onCreate(arguments)
+
+        ActivityLifecycleMonitorRegistry.getInstance().addLifecycleCallback(object : ActivityLifecycleCallback {
+            override fun onActivityLifecycleChanged(activity: Activity, stage: Stage) {
+                if (stage === Stage.PRE_ON_CREATE) {
+                    activity.window.addFlags( FLAG_KEEP_SCREEN_ON)
```

Comment:
We should add a comment explaining why this is needed. As well, what effect if any this has on simulator and other devices.